src/crystal/system/file.cr

```diff
@@ -4,7 +4,7 @@ module Crystal::System::File
   # calls.
   private def self.open_flag(mode)
     if mode.size == 0
-      raise "Invalid access mode #{mode}"
+      raise "Invalid file open mode '#{mode}'"
```

Comment:
In this branch, `mode` is an empty string. So this should probably better complain about that. At least it doesn't need to expand the variable.